scripts/latest_template_tester_report.py

```diff
@@ -28,6 +32,14 @@ def parse_cmd_line():
 
 def get_latest_tested_template_on_stream(api, template_stream_name):
     stream = {}
+    try:
+        print "wait until all the provider images are uploaded to latest directory"
+        wait_for(templates_uploaded,
+                 [api, template_stream_name], fail_condition=False, delay=5, timeout='1h')
+    except Exception as e:
+        print e
+        print "less than three provider images are uploaded to latest directory"
```

Comment:
Please, put parentheses around the print parameters since they are both python2 and python3 compatible.